cmd/rescached/rescached.cfg

```diff
@@ -49,9 +49,6 @@ server.parent.connection=udp
 ## Default:: 127.0.0.1:53
 ##
 
-server.listen=127.0.0.1:53
-
-## Uncomment line below if you want to serve rescached to other computers.
```

Comment:
Please don't delete the above line.  It was suppose to inform user that they can run rescached as server for other computers too.

Reply:
I deleted it because there is a block of documentation just above, that's why I thought it was a mistake in the first place: https://github.com/shuLhan/rescached-go/blob/master/cmd/rescached/rescached.cfg#L44